src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -96,53 +96,17 @@ namespace AirflowNetwork {
 
     enum class DuctLineType
     {
-        Invalid = -1,
+        Unknown = 0,
         SupplyTrunk,  // Supply trunk
         SupplyBranch, // SupplyBrnach
         ReturnTrunk,  // Return trunk
-        ReturnBranch, // ReturnBrnach
+        ReturnBranch  // ReturnBrnach
     };
 
-    // Using/Aliasing
-
-    // Data
-    // module should be available to other modules and routines.  Thus,
-    // all variables in this module must be PUBLIC.
-
-    // MODULE PARAMETER DEFINITIONS:
-    enum class iComponentTypeNum : int
-    {
-        Invalid = 0,
-        DOP = 1,  // Detailed large opening component
-        SOP = 2,  // Simple opening component
-        SCR = 3,  // Surface crack component
-        SEL = 4,  // Surface effective leakage ratio component
-        PLR = 5,  // Distribution system crack component
-        DWC = 6,  // Distribution system duct component
-        CVF = 7,  // Distribution system constant volume fan component
-        FAN = 8,  // Distribution system detailed fan component
-        MRR = 9,  // Distribution system multiple curve fit power law resistant flow component
-        DMP = 10, // Distribution system damper component
-        ELR = 11, // Distribution system effective leakage ratio component
-        CPD = 12, // Distribution system constant pressure drop component
-        COI = 13, // Distribution system coil component
-        TMU = 14, // Distribution system terminal unit component
-        EXF = 15, // Zone exhaust fan
-        HEX = 16, // Distribution system heat exchanger
-        HOP = 17, // Horizontal opening component
-        RVD = 18, // Reheat VAV terminal damper
-        OAF = 19, // Distribution system OA
-        REL = 20, // Distribution system relief air
-        SMF = 21, // Specified mass flow component
-        SVF = 22, // Specified volume flow component
-        Num
-    };
-
-    enum class ComponentType
+    enum class Type : int
     {
-        // TODO: enum check
-        Invalid = -1,
-        DOP = 1, // Detailed large opening component
+        Unknown = -1,
```

Comment:
Why `Unknown` and not `Invalid`, which is what we are calling that spot everywhere else?

Reply:
The ultimate goal is to eliminate additional members entirely because it isn't read in through the input processor. It's currently needed because there's a helper data structure `AirflowNetworkCompProp` that I haven't managed to get rid of yet. The helper structure is used for some introspection-type things that should instead be asking the element itself what type it is via the `type` method. I changed it to 'Unknown' because it will never be invalid and the goal it to get rid of it anyway.